internal/indexer/cmd/main.py

```
@@ -0,0 +1,244 @@
+import signal
+import os
+import sys
+from collections import Counter
+
+# Add parent directory to sys.path so 'data' and 'utils' can be imported 
+# when running `python cmd/main.py`
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+
+from data.redis_client import RedisClient
+from data.mongo_client import MongoClient
+from utils.constants import *
```

Ruff (0.15.9)

[error] `from utils.constants import *` used; unable to detect undefined names

(F403)